x-pack/plugins/ml/public/components/controls/select_severity/select_severity.js

```diff
@@ -7,65 +7,100 @@
 
 
 /*
-* AngularJS directive for rendering a select element with threshold levels.
-*/
-
+ * React component for rendering a select element with threshold levels.
+ */
 import _ from 'lodash';
+import React, { Component } from 'react';
 
-import { stateFactoryProvider } from 'plugins/ml/factories/state_factory';
+import {
+  EuiComboBox,
+  EuiHighlight,
+  EuiHealth,
+} from '@elastic/eui';
 
-import template from './select_severity.html';
-import 'plugins/ml/components/controls/controls_select';
+const OPTIONS = [
```

Comment:
these colours are hardcoded a number if times throughout our code base.
It would be nice to move them to a constants file.
`getSeverityColor` also exists to return a colour based on score, this could also be used here to avoid the hard coding of the hex values.